cfme/tests/automate/test_service_automate.py

```diff
@@ -0,0 +1,142 @@
+    yield users
+    if not BZ(1720273).blocks and user in users:
+        user.delete_if_exists()
+
+
+@pytest.fixture(scope='function')
+def infra_validate_request(domain):
+    # Take the 'ProvisionRequestApproval' class and copy it for own purpose.
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="Infrastructure"
+    ).namespaces.instantiate(name="VM").namespaces.instantiate(
+        name="Provisioning").namespaces.instantiate(name="StateMachines").classes.instantiate(
+        name="ProvisionRequestApproval").methods.instantiate(
+        name="validate_request").copy_to(domain.name)
+
+    method = domain.namespaces.instantiate(
+        name="Infrastructure"
+    ).namespaces.instantiate(name="VM").namespaces.instantiate(
+        name="Provisioning").namespaces.instantiate(name="StateMachines").classes.instantiate(
+        name="ProvisionRequestApproval").methods.instantiate(name="validate_request")
+    return method
+
+
+@pytest.fixture(scope='function')
+def service_validate_request(domain):
+    # Take the 'ProvisionRequestApproval' class and copy it for own purpose.
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="Service"
+    ).namespaces.instantiate(name="Provisioning").namespaces.instantiate(
+        name="StateMachines").classes.instantiate(
+        name="ServiceProvisionRequestApproval").methods.instantiate(
+        name="validate_request").copy_to(domain.name)
+
+    method = domain.namespaces.instantiate(
+        name="Service"
+    ).namespaces.instantiate(name="Provisioning").namespaces.instantiate(
+        name="StateMachines").classes.instantiate(
+        name="ServiceProvisionRequestApproval").methods.instantiate(
+        name="validate_request")
+    return method
+
+
+@pytest.mark.tier(3)
+@pytest.mark.provider([VMwareProvider], scope="module")
+@pytest.mark.meta(blockers=[BZ(1671563, forced_streams=['5.10'])])
+def test_user_requester_for_lifecycle_provision(request, appliance, provider, setup_provider,
+                                                new_users, generic_catalog_item,
+                                                infra_validate_request, service_validate_request,
+                                                provisioning):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: high
+        initialEstimate: 1/6h
+        tags: automate
+
+    Bugzilla:
+         1671563
+    """
+    script = """
+    user = $evm.root['user']
+    $evm.log(:info, "This is the user: #{user.userid}")
+
+    $evm.log("info", "Listing Root Object Attributes:")
+    $evm.root.attributes.sort.each { |k, v| $evm.log("info", "\t#{k}: #{v}") }
+    $evm.log("info", "===========================================")
+    """
+    infra_validate_request.update(updates={"script": script})
+    service_validate_request.update(updates={"script": script})
+
+    with new_users[0]:
```

Comment:
This `new_users` seems like a good candidate for test parametrization. Is there a reason you haven't done that here?

Reply:
Actually, test case needs two users in the system and need to login one by one.